lib/customer-deposit-wallet/customer-deposit-wallet.cabal

```diff
@@ -140,12 +149,17 @@ test-suite unit
     , cardano-wallet-test-utils
     , customer-deposit-wallet:{customer-deposit-wallet, customer-deposit-wallet-http}
     , hspec >=2.8.2
+    , hspec-golden
+    , openapi3
     , QuickCheck
     , with-utf8
+    , unliftio
```

Comment:
```suggestion
    , directory
```

I do not want to use `unliftio` — in my view, it does not sit at a good location in the design space. Essentially, monads are not a good way to structure programs in general. In a program that does not use monads for general structure, we can afford the occasional explicit call to `liftIO`. Conversely, if we cannot afford this code cost — a cost which `unliftio` can reduce — we have poor structure, and this cannot be fixed by `unliftio`.

If we want to be polymorphic in the `IO` monad, `io-classes` is a better alternative, because it allows extensive testing of concurrency.